examples/fortune/docker-compose.yml

```diff
@@ -0,0 +1,104 @@
+version: '3.7'
```

Comment:
Can we use the docker-compose in the base directory

Reply:
We can do it, but I wouldn't since it is just an example of use of hmt-escrow.
I have created a makefile to bring compile contracts, bring services up and deploy contracts with a single command like:
`make local`